packages/studioCMSBlog/src/pages/blog/index.astro

```diff
@@ -0,0 +1,34 @@
+---
+import { BlogLayout } from "studiocms-blog:layouts";
+import { PageList, RSSIcon } from "studiocms-blog:components";
+import { getPageList } from "studiocms:components";
+import config from "studiocms-blog:config";
+
+// Set the title and description
+const { description: configDescription, blogIndex } = config;
+
+const title: string = blogIndex?.title || "Blog";
+const description: string = configDescription || "Blog Index";
+const showRSSFeed: boolean = blogIndex?.showRSSFeed || true;
+
+// Get all pages
+const pageList = await getPageList();
+
+const rssUrl = `${import.meta.env.BASE_URL}rss.xml`;
```

Comment:
This interpolation is unsafe

Also, it is possible for users to change the path `rss.xml`. The final path of `rss.xml` should be retrieved using the context module:

```tsx
import { pages } from 'studiocms-blog:context';

const rssPath = pages.get('rss.xml')

if (rssPath) {
	// ...
} 
```